### DISCUSSION TITLE
Why CSS file is not part of assets in the release?

### DISCUSSION THREAD

**Discussion by r-pankevicius** [RussCoder/djvujs#47] [Q&A] [answered]
Hi,

looking at latest release:
https://github.com/RussCoder/djvujs/releases/tag/L.0.5.1_V.0.5.4

and wondering why CSS file is not an asset?

**Comment by RussCoder** [comment, marked as answer]
Hello! Since the version 0.5.0 all viewer's CSS is inside the .js file. So no separate djvu_viewer.css is provided.
